bases/shared/instagoric-server/package.json

```diff
@@ -57,5 +57,6 @@
   },
   "engines": {
     "node": ">=14.15.0"
-  }
+  },
+  "packageManager": "yarn@1.22.22+sha512.a6b2f7906b721bba3d67d4aff083df04dad64c399707841b7acf00f6b133b7ac24255f2652fa22ae3534329dc6180534e98d17432037ff6fd140556e2bb3137e"
```

Comment:
Is it intended change? Let's revert it.

Reply:
Do we intend to use yarn 1.22.22 for instagoric server?

Reply:
Which yarn version is used in entrypoint.sh to start instagoric-server?

Reply:
Adding packageManager in package.json is only useful when we use corepack  which I don't think we are using in our entrypoint.sh

Reply:
If 1.22.22 is aligned with what we use in entrypoint.sh then that us fine. Can you confirm that we already use 1.22.22 to start instagoric server in entrypoint.sh?

Reply:
![20241107_153242](https://github.com/user-attachments/assets/d089854a-3056-4749-85ce-139960b51f8d)

It is for example 1.22.19 in emerynet. Must be the same for other network/clusters.